src/components/HeaderAuth.jsx

```diff
@@ -50,10 +45,9 @@ HeaderAuth.defaultProps = {
   initialised: false,
 };
 
-function mapStateToProps(state) {  // Listens for changes in the Redux Store
-  return {
-    user: state.login.user,
-    initialised: state.login.initialised,
-  };
-}
+const mapStateToProps = (state) => ({
```

Comment:
Fair, enough. I think I've got a mismatch with my linter and PFE then, because I saw flagged...